icu4c/source/data/lang/nn.txt

```diff
@@ -2,6 +2,7 @@
 // License & terms of use: http://www.unicode.org/copyright.html
 // Generated using tools/cldr/cldr-to-icu/build-icu-data.xml
 nn{
+    %%Parent{"no"}
```

Comment:
Shouldn't some of the "nn" strings be redundant with the new "no" parent and be removed from the "nn" files?

Reply:
I think that will happen when we do the data minimization as part of CLDR BRS, that has not yet happened